app/models/session.rb

```diff
@@ -49,6 +50,8 @@ class Session < ApplicationRecord
   scope :starting_soon, -> { where("starts_at BETWEEN ? and ?", Time.current, 1.hour.from_now) }
   scope :upcoming_today, -> { where("date(starts_at) = ? and starts_at > ?", Date.current, Time.current) }
   scope :live_or_upcoming_today, -> { live.or(upcoming_today) }
+  scope :publics, -> { where(public: true) }
+  scope :privates, -> { where(public: false) }
```

Comment:
```suggestion
  scope :public, -> { where(public: true) }
  scope :private, -> { where(public: false) }
```

would opt for singular naming scopes

Reply:
I decided to use plurar scopes because singular words`private` and `public`  are reserved.

Reply:
yep, just realized that